### PR TITLE
pymysql: fix execute second argument

### LINKED ISSUE #5568: pymysql: execute() second arg
```python
import pymysql

cursor = pymysql.connect(password='hunter2').cursor()
cursor.execute("select %s as foo", "lol wat")   # type error
cursor.execute("select %s as foo", 123)   # type error
cursor.execute("select %s as foo", ["lol wat"])
```

The stub seems to say that the second argument must be tuple, list or dict. But if it isn't, pymysql escapes it correctly and everything works:

```python
    def _escape_args(self, args, conn):
        ...
        if isinstance(args, (tuple, list)):
            ...
        elif isinstance(args, dict):
            ...
        else:
            # If it's not a dictionary let's try escaping it anyways.
            # Worst case it will throw a Value error
            if PY2:
                ...
            return conn.escape(args)
```

### PR DESCRIPTION
**Pull request by Akuli**
I was going to create a `Union` of all the possibilities, but it seems to just `str()` its argument:

```
>>> cur.execute("select %s", object())
1
>>> cur.fetchone()
('<object object at 0x7f5884f04c20>',)
```

Fixes #5568

### CODE REVIEW

File: stubs/PyMySQL/pymysql/cursors.pyi [hunk 1]
```diff
@@ -21,9 +20,9 @@ class Cursor:
     def setinputsizes(self, *args) -> None: ...
     def setoutputsizes(self, *args) -> None: ...
     def nextset(self) -> Optional[bool]: ...
-    def mogrify(self, query: Text, args: _QueryArgs = ...) -> str: ...
-    def execute(self, query: Text, args: _QueryArgs = ...) -> int: ...
-    def executemany(self, query: Text, args: Iterable[_QueryArgs]) -> Optional[int]: ...
+    def mogrify(self, query: Text, args: Any = ...) -> str: ...
```
Comment (JelleZijlstra) [inline stubs/PyMySQL/pymysql/cursors.pyi:23]: Why not `object`?

**Comment by Akuli** [comment]
Just noticed that the branch for this PR somehow ended up on python/typeshed instead of Akuli/typeshed. I configured git to not let me push directly to python/typeshed.

**Comment by srittau** [comment]
Using a branch on python/typeshed is perfectly fine. In fact, it's easier for other maintainers to add changes, since they don't need to add your private repository before doing so. That said, I also use my own fork, mainly for historic reasons.

**Review by srittau — APPROVED** [review]

**Comment by JelleZijlstra** [comment]
I feel like it's preferable to use your own fork even as a maintainer so the list of branches on the main repo doesn't get polluted. You're right that it makes it a little harder for others to edit your branch though.

**Comment by Akuli** [comment]
It is possible to pass an URL directly to `git fetch`, although that's not how I do it: https://stackoverflow.com/a/58939926

**Pull request merged**